client/lib/media/store.js

```diff
@@ -17,7 +17,8 @@ import { isItemBeingUploaded } from 'lib/media/utils';
  */
 const MediaStore = {
 	_media: {},
-	_pointers: {}
+	_pointers: {},
+	_dirtyImageRef: null
```

Comment:
Wouldnt it be better if that were an array?
You can edit multiple images I think